### ISSUE TITLE
How to access training and validation losses from callbacks? 

### ISSUE THREAD

**Issue by NagarajSMurthy** [PyTorchLightning/pytorch-lightning#1930]
For example, if my validation_epoch_end in the trainer returns {'avg_loss':loss, 'log':logs}, how to get the loss value from a callback method like:def on_validation_end(trainer, pl_module)?

**Comment by MaharshiYeluri01** [comment]
you can access the current epoch variables from trainer.callback_metrics
/looks something like below
{'epoch': 4,
 'loss': tensor(0.4924, device='cuda:0'),
 'train_loss': tensor(0.4924, device='cuda:0'),
 'val_auc': tensor(0.7359, dtype=torch.float64),
 'val_loss': tensor(0.7714, device='cuda:0')}

**Comment by NagarajSMurthy** [comment]
Thank you @MaharshiYeluri01. That answers clearly.
